lib/actions/assign.js

```diff
@@ -0,0 +1,43 @@
+const { Action } = require('./action')
+
+const addAssignees = async (context, issueNumber, assignees) => {
+  return context.github.issues.addAssignees(
+    context.repo({ issue_number: issueNumber, assignees })
+  )
+}
+
+const checkAssignee = async (context, issueNumber, assignee) => {
+  const checkResult = await context.github.issues.checkAssignee(
+    context.repo({ issue_number: issueNumber, assignee })
+  ).catch(err => {
+    if (err.status === 404) return { status: 404 }
+  })
+
+  return checkResult.status === 204 ? assignee : null
```

Comment:
Would it be more readable if the function is named `isAssigneeValid` and that it returns true or false?

Reply:
sure, that make sense